Makefile

```diff
@@ -32,6 +32,7 @@ crossbuild: web
 	mkdir -p $(BUILD_DIR)/$(BIN_DIR)
 	${MAKE} build
 	cp config.mainnet.json $(BUILD_DIR)/$(BIN_DIR)/default.json
+	cp -r certs $(BUILD_DIR)/
```

Comment:
It should be `$(BUILD_DIR)/$(BIN_DIR)/certs` instead of `$(BUILD_DIR)`

Reply:
done